docs/source/client.rst

```diff
@@ -0,0 +1,38 @@
+Portal Client
+=============
+
+The ``FractalClient`` is the primary entry point to a ``FractalServer`` instance.
+
+We can initialize a ``FractalClient`` by pointing it to a server instance. If
+you would like to start your own server see the setting up a server (NYI)
+section.
+
+.. code-block:: python
+
+    >>> import qcportal as ptl
+    >>> client = ptl.FractalClient("localhost:8888")
+    >>> client
+    FractalClient(server='http://localhost:8888/', username='None')
+
+The ``FractalClient`` handles all communication to the ``FractalServer`` from
+the Python API layer. This includes adding new molecules, computations,
+collections, and querying for records.
```

Comment:
I think it would be worth adding here something to the effect of

"Fractal client without any arguments will automatically connect to the MolSSI QCArchive server if it can in read-only mode"